src/resources/SecurityCache/SecurityCacheInterfaces.ts

```diff
@@ -0,0 +1,100 @@
+export interface SecurityCacheScheduleModel {
+    enabled?: boolean;
+    frequency?: {
+        hourMinute?: {
+            hour?: number;
+            minute?: number;
+        };
+        monthDay?: number;
+        seed?: number;
+        type?: string;
+        value?: number;
+        weekDays?: [string];
+    };
+    id?: string;
+    organizationId?: string;
+    refreshType?: string;
+    resourceId?: string;
+    scheduleType?: string;
+}
+
+interface Status {
```

Comment:
should this be exported?

Reply:
I did not export It because It is not used outside this file.

Reply:
I think it should be exported, otherwise it won't get generated as type declaration in the resulting distribution. You can run `npm run build` to see if the proper types for your resource are generated in the dist folder

Reply:
Is it useful to have those interfaces to be generated as type declaration when their only goal is to build the types that will really be used by the user ?

Reply:
Yes, you can try it by yourself if you link this package to v2, then try to have autocomplete in your IDE. I think it will type it as any because it is not exported therefore not known by v2.